docs/interfaces/hmx/start_signal.md

```diff
@@ -6,11 +6,13 @@ tags:
 ---
 
 ## Overview
-There are 5 start signals on different connectors. Start 1 to 4 is meant to be used to start the device and Start 5 is meant to be used with an external LED or an output for signal start.
+There are 5 start signals on different connectors. Start 1 to 4 is meant to be used to start the device and Start 5 is meant to be used with an external LED or an output for signal start. Only one external start signal is needed to start the device. They are all connected to a a singel input and it can be read from gpio-key event or unbind gpio-key to use this signal and other digital in signals as regular gpios.
+
+**Note** unbind gpio-keys will lose the wake-up possibility on those signals and all [digital in](digital_io.md) signals.
 
 To start the device the current signal time needs to be at least 2-3 seconds long (in the future we might shorter the time) for the unit to start up and be able to hold the start signal internally.
 
-The internal signal is also used for the cut-off function.
+The internal signal (called power_on) is also used to hold the power on even if the START 1,2,3 or 4 has got down. And if you want to use the the cut-off function. Make sure that the IN_START is low(0). Before use poweroff or shutdown command.
```

Comment:
If you want to use the cut-off function, make sure that IN_START is low (0) before running poweroff or shutdown.

Reply:
```suggestion
The internal signal (called power_on) is also used to hold the power on even if the START 1,2,3 or 4 is down. 
If you want to use the the cut-off function, make sure that IN_START is low(0) before use poweroff or shutdown.
```